homeassistant/components/vera/lock.py

```diff
@@ -44,6 +48,28 @@ def is_locked(self):
         """Return true if device is on."""
         return self._state == STATE_LOCKED
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the lock."""
+        data = super().device_state_attributes
+
+        # This check will no longer be required once the version of the pyvera
+        # library with these two functions included is generally available.
+        if ("get_last_user_alert" in dir(self.vera_device)) and (
+            "get_low_battery_alert" in dir(self.vera_device)
+        ):
+            last_user = self.vera_device.get_last_user_alert()
+            if last_user is not None:
+                data[ATTR_LAST_USER_ID] = last_user[0]
+                data[ATTR_LAST_USER_NAME] = last_user[1]
+            else:
+                data[ATTR_LAST_USER_ID] = -1
```

Comment:
We don't include the attribute at all if we don't know the value.

Reply:
Sorry for the error -- I'm new to Home Assistant (and GitHub-based code reviews too...) and still learning the conventions.  Removed.

Reply:
...and now submitted.  ;-)  (Didn't realize my comment would be immediately sent to you when I entered it.  Now I know.)